python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -518,7 +526,10 @@ def run(self, sensorRef, templateIdList=None):
 
             if self.config.doMeasurement:
                 self.log.info("Running diaSource measurement")
-                self.measurement.run(diaSources, subtractedExposure)
+                if not self.config.doMeasurementNew:
+                    self.measurement.run(diaSources, subtractedExposure)
+                else:
+                    self.measurement.run(diaSources, subtractedExposure, exposure, templateExposure)
```

Comment:
Know that I'm not pushing back hard.  Just explaining.

O.K. I guess I was balking at the extra level of indirection.  I.e. the same call does different things (and more things) depending on what arguments are passed.  I would have preferred to see a call to the old measurement and then optionally a call to the new measurement.

Do we actually need to run both algorithms going forward?  The fitter seems like the "right" algorithm.